BankSDK/GiniBankSDK/Sources/GiniBankSDK/Core/ReturnAssistant/DigitalInvoice/Components/DigitalLineItemTableViewCell.swift

```diff
@@ -48,7 +48,7 @@ class DigitalLineItemTableViewCell: UITableViewCell {
             modeSwitch.onTintColor = viewModel.modeSwitchTintColor
 
             [nameLabel, editButton, unitPriceLabel, priceLabel].forEach { view in
-                view.alpha = viewModel.lineItem.selectedState == .selected ? 1 : 0.5
+                view?.alpha = viewModel.lineItem.selectedState == .selected ? 1 : 0.5
```

Comment:
[Clarification] @llevente why do we need this change?

Reply:
My XCode gives me build error that the optional "view" must be unwrapped after merging new-ui into my branch. Haven't worked on these classes, it was strange also for me. When i started working on the task it ran without error